### ISSUE TITLE
slice bounds out of range

### ISSUE THREAD

**Issue by rettier** [treeder/dockers#4]
When the VERSION file does not end in \n and is bumped from version x.x.9 to x.x.10 an exception occurs. I don't know if VERSION files without a newline are considered valid for your program, but one of my  deployment scripts recently generated such a file and i came across this error.

Steps to reproduce:

```
$ echo -n "0.2.9" > VERSION
$ docker run --rm -v "$PWD":/app treeder/bump:latest patch
Old version: 0.2.9
New version: 0.2.10
panic: runtime error: slice bounds out of range

goroutine 1 [running]:
main.bump(0xc42006e160, 0x0, 0x0)
        /go/src/github.com/treeder/dockers/bump/bump.go:98 +0x995
github.com/treeder/dockers/bump/vendor/github.com/urfave/cli.HandleAction(0x5549c0, 0x58def8, 0xc42006e160, 0x0, 0x0)
        /go/src/github.com/treeder/dockers/bump/vendor/github.com/urfave/cli/app.go:490 +0xd2
github.com/treeder/dockers/bump/vendor/github.com/urfave/cli.(*App).Run(0xc4200604e0, 0xc42000a080, 0x2, 0x2, 0x0, 0x0)
        /go/src/github.com/treeder/dockers/bump/vendor/github.com/urfave/cli/app.go:264 +0x635
main.main()
        /go/src/github.com/treeder/dockers/bump/bump.go:41 +0x3a7
```

**Comment by treeder** [comment]
Can you be sure to try the latest treeder/bump (`docker pull treeder/bump:latest`)?  This should have been fixed a bit ago.

**Comment by rettier** [comment]
Yes i am on the latest version. I think the difference to https://github.com/treeder/dockers/issues/3 is that there is no newline at the end of the file (echo **-n**)

```
$ docker pull treeder/bump                                                                            
Using default tag: latest
latest: Pulling from treeder/bump
ff3a5c916c92: Pull complete 
b9286ac91003: Pull complete 
744b00d397dd: Pull complete 
e7a5c1a486a9: Pull complete 
Digest: sha256:03f11895950762617b89bf799759a1efabbbbc16cf9764c1f221e1512b442a90
Status: Downloaded newer image for treeder/bump:latest
$ docker pull treeder/bump:1.2.5                                                          
1.2.5: Pulling from treeder/bump
Digest: sha256:03f11895950762617b89bf799759a1efabbbbc16cf9764c1f221e1512b442a90
Status: Downloaded newer image for treeder/bump:1.2.5

$ echo -n "0.2.9" > VERSION                                                                  
$ docker run --rm -it -v $PWD:/app -w /app treeder/bump patch                                         
Old version: 0.2.9
New version: 0.2.10
panic: runtime error: slice bounds out of range

goroutine 1 [running]:
main.bump(0xc420094160, 0x0, 0x0)
        /go/src/github.com/treeder/dockers/bump/bump.go:98 +0x995
github.com/treeder/dockers/bump/vendor/github.com/urfave/cli.HandleAction(0x5549c0, 0x58def8, 0xc420094160, 0x0, 0x0)
        /go/src/github.com/treeder/dockers/bump/vendor/github.com/urfave/cli/app.go:490 +0xd2
github.com/treeder/dockers/bump/vendor/github.com/urfave/cli.(*App).Run(0xc4200864e0, 0xc42000a080, 0x2, 0x2, 0x0, 0x0)
        /go/src/github.com/treeder/dockers/bump/vendor/github.com/urfave/cli/app.go:264 +0x635
main.main()
        /go/src/github.com/treeder/dockers/bump/bump.go:41 +0x3a7
```

**Comment by samarth202** [comment]
Can confirm , getting this error while going 0.0.9 > 0.0.10

**Comment by treeder** [comment]
Ok, will fix. Thanks for issue. PR welcome too. 😉

**Comment by treeder** [comment]
Should be fixed in `treeder/bump:1.2.7`. Reopen if not.
